webapp-django/crashstats/base/views.py

```diff
@@ -27,5 +26,5 @@ def handler404(request):
             'path': request.path,
             'query_string': request.META.get('QUERY_STRING'),
         }, status=404)
-    context = {'product': settings.DEFAULT_PRODUCT}
+    context = {}
     return render(request, '404.html', context, status=404)
```

Comment:
I'd leave this as is. The 404 and 500 views are different--the 404 view can take a context, but the 500 view is all "OH DESPAIR!" and can't take a context.

* https://docs.djangoproject.com/en/1.8/ref/views/#http-not-found-view
* https://docs.djangoproject.com/en/1.8/ref/views/#http-internal-server-error-view

Having said that, I have no idea if the socorro 404 handler/view/template uses the context at all. It probably does.